Dockerfile-alpine.template

```
@@ -164,7 +165,20 @@ RUN set -eux; \
 		/usr/local/share/doc \
 		/usr/local/share/man \
 	; \
-	\
+	echo '{{
+		{
+			name: "postgres",
+			version: .version,
+			params: {
+				os_name: "alpine",
+				os_version: env.variant | ltrimstr("alpine"),
+			},
+			licenses: [
+				"PostgreSQL"
+			]
+		} | sbom | tostring
+	}}' > /usr/local/bin/postgres.spdx.json \
```

```suggestion
	}}' > /usr/local/postgres.spdx.json \
```

The spdx.json file should not be put in the bin directory.